src/components/Calendar/CalendarRoot.tsx

```diff
@@ -231,6 +244,17 @@ class ScheduleCalendar extends PureComponent<ScheduleCalendarProps, ScheduleCale
             canvas.style.color = oldColor;
             calendarHeader.style.marginRight = oldMargin;
 
+            // Set text color back
```

Comment:
Again, this code block is self-explanatory with regards to _what_ it does, but I can't tell _why_ we need to reset the styles. Can you change the comment to explain the _why_, not the _what_?